CSS/inject.css

```diff
@@ -136,19 +133,16 @@ input[type="range"] {
   max-height: 95vh;
   overflow-y: scroll;
   -ms-overflow-style: none;
-  /* IE and Edge */
   scrollbar-width: none;
-  /* Firefox */
   padding: 0;
   border-radius: 5px;
   outline: 0;
   max-width: 750px;
 }
-
+/* hides scroll bar
```

Comment:
Commenting out code is bad practice. Just remove the code, it can always be recovered from Git.